.github/workflows/devbuild.yml

```diff
@@ -667,7 +667,7 @@ jobs:
     needs: [standalone_buffer, build_ubuntu, build_macos, build_windows]
     runs-on: ubuntu-latest
     # Run if any of the dependencies failed in master branch
-    if: ${{ always() && contains(needs.*.result, 'failure') && github.ref_name == 'master' }}
+    if: ${{ always() && contains(needs.*.result, 'failure') && github.ref_name == 'master' && github.event.repository.fork == false }}
```

Comment:
Beautiful.

Maybe we should make it multi-line for easier reading.  Long lines sometimes get hard to read.

But this length is still tolerable.  Let's worry about it in the future.